backend/backend/services/business_logic.py

```diff
@@ -13,7 +13,7 @@ async def get_test_result(context: Context, subject: str):
     summary_chain = context.chains.get_chains()['summary_chain']
 
     knowledge = []
```

Comment:
<begin comment>Consider reducing the range in this loop to improve performance.<end comment>